website/docs/r/codebuild_project.html.markdown

```diff
@@ -150,6 +150,7 @@ resource "aws_codebuild_project" "example" {
     type            = "GITHUB"
     location        = "https://github.com/mitchellh/packer.git"
     git_clone_depth = 1
+    source_version = 'master'
```

Comment:
I'm actually not sure if single quotes are valid 🤔 , but common convention in Terraform configurations is double quotes.

```suggestion
    source_version  = "master"
```